google-cloud-spanner/lib/google/cloud/spanner/batch_client.rb

```diff
@@ -272,6 +272,77 @@ def load_partition serialized_partition
           Partition.load serialized_partition
         end
 
+        ##
+        # Creates {Fields} object from types.
+        #
+        # This object can be used to create {Data} objects by providing values
+        # that match the field types. See {Fields#struct}.
```

Comment:
How about:

This method returns a configuration object ({Fields}) that may be provided to queries or may be used to create STRUCT objects. (The STRUCT is represented by the {Data} class.) See {Client#execute} and/or {Fields#struct}.

Reply:
Updated:

This method returns a configuration object ({Fields}) that may be provided to queries or used to create STRUCT objects. (The STRUCT **will be** represented by the {Data} class.) See {Client#execute} and/or {Fields#struct}.

Reply:
Fixed.